frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -67,6 +67,7 @@ export type Hyperparameter = {
     type: HyperparameterType;
     typeOptions: HyperparameterType[];
     scalingType?: string;
+    required: boolean;
```

Comment:
Couldn't we just default this to false so we don't have to update all the fields below and just the ones we want to be required:

`required: boolean = false;`

Reply:
Doesn't seem like I can do defaults like that: https://stackoverflow.com/questions/61862235/default-value-for-typescript-type-alias

I *could* set the default like this:
```
// Declare the type
export type SomeType = {
    typename: string;
    strength: number;
    radius: number;
    some_func: Function;
    some_other_stat: number;
}

// Create an object with all the necessary defaults
const defaultSomeType = {
    some_other_stat: 8
}

// Inject default values into your variable using spread operator.
const someTypeVariable: SomeType = {
  ...defaultSomeType,
  typename: 'name',
  strength: 5,
  radius: 2,
  some_func: () => {}
}
```

Reply:
Interesting, I thought you could do this. No worries! I'll approve.